### PR TITLE
Java 11

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -178,6 +178,20 @@
...
     </dependencyManagement>
 
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <configuration>
+                        <release>8</release>
```
Comment (vilchik-elena) [inline pom.xml]: why we need this?

File: Jenkinsfile [hunk 2]
```diff
@@ -61,8 +61,10 @@ pipeline {
                         label 'windows'
                     }
                     steps {
-                        withMaven(maven: MAVEN_TOOL) {
-                            sh 'mvn.cmd clean test'
+                        withQAEnv {
```
Comment (vilchik-elena) [inline Jenkinsfile]: why do we need QA env for CI?

**Comment by vilchik-elena** [comment]
@saberduck probably we should not squash this PR, how about splitting it in 2 commits?

File: pom.xml (hunk 1 above)
Comment (saberduck) [reply pom.xml]: This is to make sure that generated artifact can be used with Java 8

File: Jenkinsfile (hunk 2 above)
Comment (saberduck) [reply Jenkinsfile]: it's setting the maven and java version

**Pull request merged**